docs/platform/webcontainers/browser-support.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: &title WebContainers Browser Support
-description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, and Firefox in alpha state.
+description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, Safari 16.4 TP, and Firefox in alpha state.
```

Comment:
I'm not that good at comma's, but do we need to remove this one to indicate that Safari is also alpha?

```suggestion
description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, Safari 16.4 TP and Firefox in alpha state.
```

Reply:
Oh I see. 

```suggestion
description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in alpha state.
```

Reply:
Yes, I like that!